src/start/RealmScreen.js

```diff
@@ -90,7 +83,7 @@ class RealmScreen extends React.Component {
               autoCapitalize="none"
               placeholder="Server URL"
               defaultValue={realm}
-              onChangeText={value => this.setState({ realm: value })}
+              onChangeText={value => this.setState({ realm: fixRealmUrl(value) })}
```

Comment:
Yeah , I know this. Don't know why it's happening. That's why I asked if it's a good way to do it?
Apart from this, anything you write starts with `https://` Just this thing is lagging.